_projects/ashlar.md

```diff
@@ -0,0 +1,42 @@
+---
+published: false # change to true once ready to post
+
+title: "Ashlar" # project title inside quotes
+excerpt: "Ashlar schema editor in Django admin" # shows on project list page
+seo_description: "" # goes in project meta description
+
+# project attributes
+requirements: # bullet list of requirements – one requirement per line – follow below format
+  - "Python experience"
+  - "Django experience preferred"
+  - "Javascript experience preferred"
+tags: # one tag per line – spaces are allowed in tags – follow below format
+  - "python"
+  - "django"
+  - "jsonb"
+  - "postgresql"
+difficulty: "hard" # easy, medium, hard – pick one
+mentors: # github username without the @ – example: designmatty
+  - "ddohler"
+  - ???
+
+# This file uses Kramdown. See https://kramdown.gettalong.org/syntax.html for syntax
+---
+
+# MILESTONES
+
+## PREPARATION
+
+- Familiarization with Ashlar project
+- 
+
+## CODING PHASE 1
+
+- Bring Ashlar project dependencies up to date
+- Support Python 3 (?)
```

Comment:
~(?)~
I don't think it makes sense to put work into updating it but not support Python 3.